jenkins-integration/beaker/install/foss/30_install_dev_repos.rb

```diff
@@ -46,6 +46,25 @@ def get_latest_master_version(branch)
 end
 
 def get_latest_agent_version
+  url = "https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent%20suite%20pipelines/job/platform_puppet-agent_intn-van-promote_suite-daily-promotion-master/lastSuccessfulBuild/api/json"
```

Comment:
This is the same approach we would be using in OSS puppetserver to determine the latest puppet-agent version per https://github.com/puppetlabs/puppetserver/pull/1432.